### DISCUSSION TITLE
The full code link of the Iframe Plugin example failed

### DISCUSSION THREAD

**Discussion by xiaomingdedaima** [Milkdown/milkdown#874] [General]
At the end of **Example: Iframe Plugin**, the **full code** is no longer valid and needs to be updated, thanks
Could you please provide a new link for reference first?
![image](https://user-images.githubusercontent.com/105727662/212230154-1919ba72-5e95-4836-a158-7c61f0d494c5.png)

**Comment by Saul-Mirone** [comment]
Thanks for reporting, we're fixing that.
